menu/drivers/ozone.c

```diff
@@ -562,6 +564,13 @@ struct ozone_handle
    char tab_path[PATH_MAX_LENGTH];
    char fullscreen_thumbnail_label[255];
 
+   /* These have to be huge, because runloop_st->name.savestate
+    * has a hard-coded size of 8192...
+    * (the extra space here is required to silence compiler
+    * warnings...) */
+   char savestate_thumbnail_file_path[8204];
```

Comment:
I wonder if we can eventually vastly reduce the size of runloop_st->name.savestate. Perhaps it's best if we work with base paths and only put together the 'full path' right before a file I/O call.

Reply:
Yep, that current way does not seem optimal, but since it is like that in XMB, I didn't give it too much thought for now.